source/develop/release-management/features/engine/separate-reports-host.html.md

```diff
@@ -436,7 +436,7 @@ Some installations might want to separate for security reasons, e.g. to give som
 
 ## Presentation
 
-![](Engine-dwh-reports-on-separate-hosts-presentation.zip "fig:Engine-dwh-reports-on-separate-hosts-presentation.zip") pandoc source and the generated html included.
+[ZIP](/images/wiki/Engine-dwh-reports-on-separate-hosts-presentation.zip) pandoc source and the generated html included.
```

Comment:
Note: I still think it's wrong and odd to have non-image content under images. (It should reside elsewhere.)